### DISCUSSION TITLE
Github student developer pack

### DISCUSSION THREAD

**Discussion by Saicharan2001** [github/india#447] [Students]
Hey hi guys 
This is Saicharan actually I have something to get clarified , I redeemed student developer pack a long back for some other purposes and I have only 41 USD left , now can i proceed with this internship smoothly because we need to have certain credits for using minimum of three azure services for our project
Would I able to move on smoothly with the rest credits left in the Azure students subscription from github developer pack ??
Could someone reply please

**Comment by Prathamesh248** [comment]
unable to claim github student pack

**Comment by Saicharan2001** [comment]
You are supposed to claim it using your claim email id
Don't try to claim through the FRT website it's waste of time
If u use your college email id and if you are fortunate enough you will
availing the pack in few seconds

**Comment by Prathamesh248** [comment]
Can I use my email . I tried using my email but it's not working.
I unable to use my clg email its asking for password. What should I do?. I
also uploaded my id.

**Comment by Saicharan2001** [comment]
You cant use your personal email dude !!!
You are expected to use only  email id provided by your college email which
includes your college short form or something like that ..
Btw which clg are you from ??

**Comment by Prathamesh248** [comment]
Bro I'm from Walchand Institute of Technology Solapur. My clg email is
***@***.*** but it's not working asking for verifying email id

**Comment by Saicharan2001** [comment]
Bro I got it done in seconds , through my clg email id
Just check this out on YouTube

**Comment by Prathamesh248** [comment]
I watched some videos don't find helpful.
Help me
